typings/index.d.ts

```diff
@@ -2422,7 +2424,9 @@ declare module 'discord.js' {
 
   interface ChannelPosition {
     channel: ChannelResolvable;
-    position: number;
+    lockPermissions?: boolean;
+    parent?: ChannelResolvable;
```

Comment:
Why isn't this `CategoryChannelResolvable`?

Reply:
no such thing exists, all current instances use ChannelResolvable, CategoryResolvable should be introduced after all the stuff that'd need it lands in one central PR, imo

Reply:
Resolved in c732669d49713efa707fe5df9731028f40ffbc2e for this PR only, #5521 to keep track of and apply the type to the rest of the library